requirements/customer/glossary.tex

```diff
@@ -0,0 +1,83 @@
+\newglossaryentry{ID}
+{
+  name=ID,
+  plural=IDs,
+  description={
+    An ID is a string of digits, english letters (both lower- and uppercase) and slashes '-'.
+  }
+}
+\newglossaryentry{job status}
+{
+  name=job status,
+  plural=job status,
+  description={
+    The status of a job can be:
+    \begin{itemize}
+      \item Queued - the job is waiting for a suitable work machine to become available.
+      \item Running - the job is running on some work machine.
+      \item Paused - the job has been running, but has been paused to allow for execution of other tasks.
+      \item Done - the job has finished running.
```

Comment:
Killed - the process was forcefully stopped.
Crashed - the work machine the job was running on shut down unexpectedly.